### ISSUE TITLE
USER ISSUE: Consolidate Inventories crash

### ISSUE THREAD

**Issue by madcatandrew** [StrangeLoopGames/EcoIssues#23102]
\
**User Id: (Not logged user)**
**Version:** 0.9.5.0 beta release-301

**Steps to Reproduce:**
Accessed large lumber stockpile, various chests nearby and powered cart. Clicked consolidate inventories, crashed.

**Expected behavior:**
Consolidated I guess.

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: DivideByZeroException
Message:Attempted to divide by zero.
Source:Eco.Gameplay

System.DivideByZeroException: Attempted to divide by zero.
   at Eco.Gameplay.Items.InventoryChangeSet.ConsolidateInventory(Inventory inventory, User user)
   at Eco.Gameplay.Components.LinkComponent.ConsolidateLinkedInventory(User user)
Outer Exceptions:
Exception has been thrown by the target of an invocation.
Exception has been thrown by the target of an invocation.</size>
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No mods

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by madcatandrew** [comment]
After rejoin, and persistent across server restarts, many benches, doors and chests near the stockpile which caused the crash are now completely invisible, but still accessible from other inventories in range.

Rejoining the server causes the exact benches and chests which are invisible to change to different ones.

**Comment by SlayksWood** [comment]
@madcatandrew Thank you for the report. 
Can you provide log files after crash. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by SlayksWood** [comment]
@madcatandrew Can you take a couple of screenshots of your storages before clicking the button?

**Comment by madcatandrew** [comment]
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/8583574/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/8583575/Player-prev.log)
![20220428082618_1](https://user-images.githubusercontent.com/1343830/165775376-b5eee410-eef5-4a7e-960c-c80cd9517c0d.jpg)
![20220428082636_1](https://user-images.githubusercontent.com/1343830/165775379-d227caf4-bb05-4ab1-9131-646c579ed911.jpg)
![20220428082649_1](https://user-images.githubusercontent.com/1343830/165775380-48dea01a-6286-4ca9-802f-6add6b005a47.jpg)
![20220428082657_1](https://user-images.githubusercontent.com/1343830/165775384-bf2164a3-6634-4f04-86f9-2139bd1213be.jpg)
![20220428082701_1](https://user-images.githubusercontent.com/1343830/165775387-0b613038-8615-4a6a-a01b-efbff985450a.jpg)

**Comment by madcatandrew** [comment]
Leaving my server backup pre-update here to reproduce from. Backup was about 60m from house, looking towards it. Reproduction of the error reliably happens upon consolidating with any container near my house.

Also including new log files since I can reproduce it anytime, and I think those prior logs were not from an active crash.
This one definitely includes it.

[World.zip](https://github.com/StrangeLoopGames/EcoIssues/files/8587063/World.zip)
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/8587085/Player.log)

**Comment by D3nnis3n** [comment]
Should be fixed in 9.5.1, please reopen if it reoccurs.
